foreman_dlm.gemspec

```diff
@@ -11,16 +13,16 @@ Gem::Specification.new do |s|
   # also update locale/gemspec.rb
   s.description = 'Adds a Distributed Lock Manager to Foreman. This enables painless system updates for clusters.'
 
+  s.required_ruby_version = '>= 2.5', '< 4'
```

Comment:
You can raise it to 2.7+

Reply:
Need to keep it at 2.5 because of https://github.com/theforeman/theforeman-rubocop/blob/main/rules/ruby_target.yml

Reply:
We need to update that. In the short term, you can override it in your local RuboCop config.